src/fiatExchanges/quotes/ExternalQuote.ts

```diff
@@ -73,11 +73,15 @@ export default class ExternalQuote extends NormalizedQuote {
   }
 
   getFeeInFiat(_exchangeRates: { [token in Currency]: string | null }): BigNumber | null {
-    return isSimplexQuote(this.quote)
-      ? new BigNumber(this.quote.fiat_money.total_amount).minus(
-          new BigNumber(this.quote.fiat_money.base_amount)
-        )
-      : new BigNumber(this.quote.fiatFee)
+    if (isSimplexQuote(this.quote)) {
+      return new BigNumber(this.quote.fiat_money.total_amount).minus(
+        new BigNumber(this.quote.fiat_money.base_amount)
+      )
+    } else if (typeof this.quote.fiatFee === 'number') {
+      // Can't just check for truthiness since '0' fails this and introduces a regression
```

Comment:
I'm a bit confused by this comment in part because I can't figure out the scenario in which a value of '0' is received. Aren't the quotes supposed to return a number from the [cloud function](https://github.com/valora-inc/cloud-functions/blob/main/src/cico/fetchProviders.ts#L40) for fiatFee (assuming they're not a simplex quote)?

Reply:
I'm not sure it's impossible for `fiatFee` to be `0`; I assumed there was a possibility that could happen; if that's the case, and it's possible for our cloud function to return the number `0`, then just checking truthiness would introduce a regression here. (This was actually caught by one of our existing unit tests, which uses mock data where the `fiatFee` was already set to `0`.)

Reply:
nit: can you use backtick instead of single quote for 0, I originally read this as a string containing 0